python/lsst/ts/observatory/control/auxtel/latiss.py

```diff
@@ -301,7 +324,9 @@ def usages(self):
                 components_attr=["atspectrograph"],
                 readonly=False,
                 generics=["summaryState"],
-                atspectrograph=["changeFilter", "changeDisperser", "moveLinearStage"],
+                atspectrograph=["changeFilter", "changeDisperser", "moveLinearStage",
```

Comment:
Can you add this to the `self.valid_use_cases.All` and `self.valid_use_cases.TakeImageFull` usages?